### DISCUSSION TITLE
Welcome to Hyperion Discussions!

### DISCUSSION THREAD

**Discussion by Matt-Caspermeyer** [Matt-Caspermeyer/Hyperion#1] [Announcements]
## 👋 Welcome to the discussion area for the EFS MOD Hyperion!
  I'm using Discussions as a place to connect with other members of the EFS MODding community. I hope that you:
  * Ask questions about Hyperion.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded.
  * Remember that this is a community we build together 💪.

My name is Matt Caspermeyer and I am the original author of the Hyperion MOD for EFS and have been an integral member of the EFS V1.5 & V1.51 development teams.

**Comment by psilocybegames** [comment]
hey Matt, I played a TON of Hyperion back in the late 90s and I appreciate your work so much. I'd like to contribute, I'm a game developer with years of experience. What can I do?

**Reply by Matt-Caspermeyer** [reply]
> hey Matt, I played a TON of Hyperion back in the late 90s and I appreciate your work so much. I'd like to contribute, I'm a game developer with years of experience. What can I do?

Thanks for your interest!

The short of it is that I need people to start playing it!

Here's a more detailed description of what I need help with...

The MOD is currently in the Alpha stage, but it is very playable - I'm certainly looking for help with the artwork, especially the 32 x 32 pixel "chiclet" icon art. I have about 99% of the artwork for all the current units, but I have trouble with reducing the pictures, from 512 x 512 (for example) down to 32 x 32, without them being blurry or too pixelated and since I'm colorblind, I have a lot of trouble with making it look good.

So that's one area that I need help in - I have all the original artwork and can provide a link if you want to help out in this area.

If that is not your forte, I just need you to start playing and providing feedback.

I'm currently working on the Hyperion Historical Galaxy - this is an update to the original Historical Galaxy from standard EFS for Hyperion.

I'm looking for feedback from the playtesting viewpoint of that galaxy for each of the 5 Houses.

I'm also doing a lot of playtesting myself right now, and things that I need to work on are ensuring that there is sufficient unit delineation of the early units so that their statistics are not almost the same. I believe that I have largely been successful in this area, but it is possible that I've missed some outliers here and need to find them, still.

I have a formulaic approach to unit design, and basically a significant number of units have their statistics automatically generated. Sometimes, I don't have sufficient research distance between a Jeep and Jeep II, for example, and the Jeep II's statistics are automatically generated from the Jeep's statistics, but if there is almost the same technology input into Jeep II's as there is for Jeep's then the Jeep II's statistics will be almost the same. So, I just fixed that issue yesterday and I'm just looking to check all this kind of stuff out. Note that I cannot change the individual statistics for autogenerated units, but I can for the base units, of which they scale, and so I have a point system that determines if the unit has been designed correctly. I'm in the process of checking all this out, right now. If you are playing and you note that unit A almost has the same stats as unit B, then let me know and I can then look into it.

The rest is pretty much in the README document, but I need people to play several decades worth of turns to see where there are unit gaps so that they can start to be filled in with missing units.

That's my plan for this year is to fill in missing units, add any missing technologies (although I estimate the tech tree to be 99% complete), and then start completing the Manowitz volumes.

**Reply by psilocybegames** [reply]
Awesome. I'd take a look at the art first. Could we move the conversation to email or Discord?

**Reply by Matt-Caspermeyer** [reply]
> Awesome. I'd take a look at the art first. Could we move the conversation to email or Discord? ![image](https://private-user-images.githubusercontent.com/38584440/435488962-28c5a472-5bbf-435a-ad11-1ae1f947c0ba.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3NDUxNzUwMzAsIm5iZiI6MTc0NTE3NDczMCwicGF0aCI6Ii8zODU4NDQ0MC80MzU0ODg5NjItMjhjNWE0NzItNWJiZi00MzVhLWFkMTEtMWFlMWY5NDdjMGJhLnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNTA0MjAlMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjUwNDIwVDE4NDUzMFomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPWMzOWNlMTRkYjg5YzYzYjgxNjhkMWM2MWY1NDU5ZDJjMzVhNDY5MmEzNzU2OTk1NWQxZDhhYzUwYzE4YzkwNTgmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0In0.F7lcghiCHOBkTMk2Az0RQSpgMRLxevRxTtOzhQ-wE4k) or ![image](https://private-user-images.githubusercontent.com/38584440/435488986-3d525fd0-10e5-4817-998a-ee4f79cbfeb3.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3NDUxNzUwMzAsIm5iZiI6MTc0NTE3NDczMCwicGF0aCI6Ii8zODU4NDQ0MC80MzU0ODg5ODYtM2Q1MjVmZDAtMTBlNS00ODE3LTk5OGEtZWU0Zjc5Y2JmZWIzLnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNTA0MjAlMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjUwNDIwVDE4NDUzMFomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPWJkNjRiYTlkZTFiZmNlYzMzNTBkZWI0YjMxYTdhYWUzMWM4MTU0N2RhOTgxMzM3OWU0ZjJlZTY0Nzc2NjYyNTgmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0In0.2xlHCXr_bJc8zeHRXL3U3ibzk2yCEjf4c4yMT4Yy-n0) on DC

Okay, sounds great!

I'm on Discord "Matt Caspermeyer" and I'll send you an email as well!
